extensions/adobe/experience/primetime-authentication/application.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":   "https://ns.adobe.com/experience/primetime-authentication/application",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Application extensions for Adobe Primetime Authentication",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/application"],
+  "description":
+    "Application extension properties specific to Adobe Primetime Authentication.",
+  "definitions": {
+    "application": {
+      "properties": {
+        "xdm:clientAPIVersion": {
+          "title": "Client API Version",
+          "type": "string",
+          "description":
+            "The client API version embedded or used by the application."
```

Comment:
What client? Primetime? Improve the description please so it is clear what is supposed to go in here.

Reply:
Client here stands for "Primetime Authentication SDK". Maybe we should call it that. @cdobre ?

Reply:
I have removed this field for now. We need to discuss internally if is important to track various sdk versions or server versions.